HWI/prompt_assignment.py

```
@@ -0,0 +1,155 @@
+# 
+# - Role Prompting (4개 이상 역할)
+# - Few-shot (예시 1, 3)
+# - CoT (단계적 사고)
+# - Temperature/Top-p 스윕 (3회 반복)
+# - Prompt Injection 내성 테스트
+# - 결과 자동 로깅 및 CSV 저장
+
+# In[31]:
+
+
+get_ipython().system('pip -q install openai python-dotenv pandas numpy nltk matplotlib')
```

Ruff (0.13.1)

Undefined name `get_ipython`

(F821)